applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -109,6 +110,16 @@ def __build_cmd_get(core: Core, device: AudioDevice, build: BuildType, pristine,
         if options.nrf21540:
             device_flag += " -DSHIELD=nrf21540ek_fwd"
 
+        if options.user_bt_name:
+            MAX_USER_NAME_LEN = 248
+            user_specific_bt_name = ("AUDIO_DEV_" + getpass.getuser())[:MAX_USER_NAME_LEN].upper()
+            device_flag += " -DCONFIG_BT_DEVICE_NAME=\\\"" + user_specific_bt_name + "\\\""
+
+        if options.custom_bt_name != None:
+            MAX_USER_NAME_LEN = 248
```

Comment:
I know I have defined this variable twice. I thought this was better than to have only one declaration that is far from where it is used. But I can change this if people want

Reply:
I think it's better to define this once at the top of the file. Variables in capital letters are expected to be global constants, so people will know where to look for it.